.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+### Features Description
+1. Feature explanation
+   - Feature specification if you want to add
```

Comment:
I prefer to remove example text

Reply:
I like the example, it is easier to see where you should put the text, then just letting in blank the numbers and the - for example

Reply:
Ok. We can keep it.